cli/Dockerfile

```diff
@@ -38,12 +38,12 @@ RUN export DEBIAN_FRONTEND=noninteractive \
           tee /etc/apt/sources.list.d/hashicorp.list \
      && apt update && apt-get install terraform \
      # kubectl CLI
-     && apt-get update \
-     && mkdir -p /etc/apt/keyrings \
-     && curl -fsSLo /etc/apt/keyrings/kubernetes-archive-keyring.gpg https://dl.k8s.io/apt/doc/apt-key.gpg \
-     && echo "deb [signed-by=/etc/apt/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | tee /etc/apt/sources.list.d/kubernetes.list \
-     && apt-get update \
-     && apt-get install -y kubectl \
+     && curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" \
+     && curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl.sha256" \
+     && echo "$(cat kubectl.sha256)  kubectl" | sha256sum --check \
+     && install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl \
+     && kubectl version --client \
+     && kubectl version --client --output=yaml \
```

Comment:
do we need both?

Reply:
Just follows the docs - doesn't hurt anybody.